chingu-voyages/v19-bears-team-07
# Edit dev

This pull request implements a page to edit the profile of the logged-in user at `/profile-page`. It also changes the authentication endpoint in Rails from `/users` to `/auth`.

To test the editing functionality:

- Log in using your account.
- Visit `/profile-page` using the navigation bar.
- Fill in the form.
- Submit the form
- Refresh the page. All your information should still be present,

![profile](https://user-images.githubusercontent.com/62522740/82106148-1eab5700-96d4-11ea-9576-03e936f553e0.PNG)
